### PR TITLE
Added fixed for address location bug #165

### PR DESCRIPTION
**Pull request by TenjouUtena**
Added a fix for #165

### CODE REVIEW

File: src/client/modules/main/commands/address/Address.js [hunk 1]
```diff
@@ -112,8 +112,16 @@ class Address {
 		let charIds = params.charIds || [];
 		if (!charIds.length) {
 			charIds = this.lastCharIds[char.id];
+			let inRoom = true;
+			for (let c of charIds) {
+				if (!this.module.cmdLists.getInRoomCharsAwake().getItemByID(c)) {
```
Comment (ItsAKelmi) [inline src/client/modules/main/commands/address/Address.js:117]: Minor note, but would be more efficient to create the `CharList` via `this.module.cmdLists.getInRoomCharsAwake()` before the loop only once.

**Comment by TenjouUtena** [comment]
Refactored

**Comment by anisus** [comment]
While the PR does solve the issue on the client, I think this is a backend bug really that should be handled there. Similar to how we do it for `whisper`. Sorry for not commenting on Talon's PR previously!

**Comment by TenjouUtena** [comment]
> While the PR does solve the issue on the client, I think this is a backend bug really that should be handled there. Similar to how we do it for `whisper`. Sorry for not commenting on Talon's PR previously!

Closing per Acci's notes in

**Pull request closed without merging**